api_tests/files/views/test_file_detail.py

```diff
@@ -634,9 +635,9 @@ def file(self, root_node, user):
             'service': 'cloud',
             osfstorage_settings.WATERBUTLER_RESOURCE: 'osf',
         }, {
-            'size': 1337,
-            'contentType': 'img/png'
-        }).save()
+                                'size': 1337,
+                                'contentType': 'img/png'
+                            }).save()
```

Comment:
This changes seems off. In addition, I don't think we need to update the style for files we don't touch. But no worries, I will take care of this and similar style stuff before/during merge.